gen-go/agent/common/v1/common.pb.go

```diff
@@ -1,25 +1,47 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 // Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.25.0
+// 	protoc        v3.12.3
 // source: opencensus/proto/agent/common/v1/common.proto
 
+// NOTE: This proto is experimental and is subject to change at this point.
+// Please do not use it at the moment.
+
 package v1
 
 import (
-	fmt "fmt"
 	proto "github.com/golang/protobuf/proto"
```

Comment:
Apparently new generator still uses `"github.com/golang/protobuf/proto"` just not for marshaling.